.github/workflows/pypi-publish.yml

```diff
@@ -22,6 +22,7 @@ jobs:
 
     - name: Install twine
       run: >-
+        python -m pip install --upgrade pip
```

Comment:
[really nitpick]: the next line uses `install -U`, so how about unifying them for consistency?